buildscripts/semantic-convention/generate.sh

```diff
@@ -3,9 +3,9 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOT_DIR="${SCRIPT_DIR}/../../"
 # freeze the spec version to make SemanticAttributes generation reproducible
-# this hash was obtained by calling `git rev-parse master`
+# this hash was obtained by calling `git rev-parse master` on 1st Dec 2020
 # we can't use a version tag here because otel-spec releases are very rare
```

Comment:
fyi I brought this up in the spec meeting this morning, so it's at least on people's radar.